package failure

import "testing"

func TestDirigentFailure(t *testing.T) {
	//triggerDirigentFailure("cvetkovi@pc846.emulab.net", "control_plane", 0)
	triggerDirigentFailure("cvetkovi@hp044.utah.cloudlab.us cvetkovi@hp013.utah.cloudlab.us cvetkovi@hp052.utah.cloudlab.us cvetkovi@hp093.utah.cloudlab.us cvetkovi@hp066.utah.cloudlab.us cvetkovi@hp189.utah.cloudlab.us cvetkovi@hp065.utah.cloudlab.us cvetkovi@hp083.utah.cloudlab.us cvetkovi@hp076.utah.cloudlab.us cvetkovi@hp099.utah.cloudlab.us cvetkovi@hp060.utah.cloudlab.us cvetkovi@hp056.utah.cloudlab.us cvetkovi@hp144.utah.cloudlab.us", "data_plane", 0)
	//triggerDirigentFailure("cvetkovi@pc727.emulab.net", "worker_node", 0)
}
